pkg/bazel/bazel.go

```diff
@@ -58,23 +60,31 @@ func WriteBuild(dryRun bool, buildfile *build.File, path string) error {
 		fmt.Println(build.FormatString(buildfile))
 		return nil
 	}
-	return ioutil.WriteFile(path, build.Format(buildfile), 0666)
+	return os.WriteFile(path, build.Format(buildfile), 0666)
 }
 
 func WriteWorkspace(dryRun bool, workspace *build.File, path string) error {
 	if dryRun {
 		fmt.Println(build.FormatString(workspace))
 		return nil
 	}
-	return ioutil.WriteFile(path, build.Format(workspace), 0666)
+	return os.WriteFile(path, build.Format(workspace), 0666)
 }
 
 func WriteBzl(dryRun bool, bzl *build.File, path string) error {
 	if dryRun {
 		fmt.Println(build.FormatString(bzl))
 		return nil
 	}
-	return ioutil.WriteFile(path, build.Format(bzl), 0666)
+	return os.WriteFile(path, build.Format(bzl), 0666)
+}
+
+func WriteLockFile(config *bazeldnf.Config, path string) error {
+	configJson, err := json.Marshal(config)
+	if err != nil {
+		return err
+	}
+	return os.WriteFile(path, configJson, 0666)
```

Comment:
Shouldn't this be 0644?

Reply:
Not sure if this is big deal @kellyma2 but would be nice to have it, we can fix it after merge

Reply:
I think 0644 is fine, but I was just being consistent with the other instances in here.